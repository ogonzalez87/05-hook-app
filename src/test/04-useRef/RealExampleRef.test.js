import { shallow } from "enzyme";
import React from "react";
import { RealExampleRef } from "../../components/04-useRef/RealExampleRef";

describe("RealExampleRef", () => {
  const wrapper = shallow(<RealExampleRef />);
  test("should show", () => {
    expect(wrapper).toMatchSnapshot();

    expect(wrapper.find("MultipleCustomHooks").exists()).toBe(false);
  });
  test("should show component", () => {
    const btn = wrapper.find("button");
    btn.simulate("Click");
    console.log(wrapper.find("MultipleCustomHooks").exists());
  });
});
