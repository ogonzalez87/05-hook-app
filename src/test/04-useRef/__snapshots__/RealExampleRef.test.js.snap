// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RealExampleRef should show 1`] = `
<div>
  <h1>
    RealExampleRef
  </h1>
  <hr />
  <button
    className="btn btn-primary mt-5"
    onClick={[Function]}
  >
    Show/Hide
  </button>
</div>
`;
